pitest-entry/src/test/java/org/pitest/mutationtest/config/CompoundListenerFactoryTest.java

```diff
@@ -52,12 +50,8 @@ public void setUp() {
   public void shouldCreateACombinedListenerForAllChildFactories() {
     final MutationResultListener listenerOne = mock(MutationResultListener.class);
     final MutationResultListener listenerTwo = mock(MutationResultListener.class);
-    when(
-        this.firstChild.getListener(any(Properties.class),
-            any(ListenerArguments.class))).thenReturn(listenerOne);
-    when(
-        this.secondChild.getListener(any(Properties.class),
-            any(ListenerArguments.class))).thenReturn(listenerTwo);
+    when(this.firstChild.getListener(isNull(), isNull())).thenReturn(listenerOne);
```

Comment:
@AlexElin would you mind improving this test? You're completely right in changing the expectation from any() to isNull() but this reveals that the test is ... incomplete at least...

I think we should add another test in order to check that the non-null arguments provided to  testee will be forwarded to first and second child. Do you agree?

Reply:
`isNull` is rarely the right matcher. If null is being passed around it is most likely an artefact of the tests, rather than part of any contract.

Generally, stubs should be as general and accepting as possible (so any(Class) rather than restricting to null values). 

Verifications should be as precise as necessary (so rarely use any, unlikely to us isNull).

Reply:
changed.
Passed non-null arguments to `this.testee.getListener(...)`